prow/github/hmac.go

```diff
@@ -46,3 +64,37 @@ func PayloadSignature(payload []byte, key []byte) string {
 	sum := mac.Sum(nil)
 	return "sha1=" + hex.EncodeToString(sum)
 }
+
+// For given repository/organization, returns all *valid* HMAC tokens. Validity is determined based on time till they are valid.
+func ExtractHmacs(repo string, tokenGenerator func() []byte) [][]byte {
```

Comment:
Is this public for a specific reason?

Reply:
Nope, it didn't need to be. Making private.